### DISCUSSION TITLE
Carga de datos desde github

### DISCUSSION THREAD

**Discussion by galvanic90** [rramosp/ai4eng.v1#301] [UDEA Students]
Tengo la siguiente duda, otra opción para cargar los datos es hacerlo desde el repositorio de github en el cual vamos a alojar los notebooks, de esa manera no sería necesario subir a colab el token de autenticación de Kaggle. Lo puedo hacer? o profe prefiere que los cargue desde Kaggle?

**Comment by Felipe-RA** [comment]
Saludos @galvanic90 

Github es un sistema de versionamiento de código, como tal no se recomienda que cargues datos en el. Como mucho imágenes o pdfs con el objetivo de realizar documentación.

En su lugar, lo ideal es que se incorpore a los notebooks la utilización de una API para descargarlos a través de ella (aunque en el caso del proyecto del ICFES pueden suponer que tendremos los datos en local cuando vayamos a evaluar).

OJO, eso sí, aclarar muy bien en dónde espera vuestro proyecto que se alojen estos datos y nada usar **rutas absolutas**, siempre **[rutas relativas](https://phoenixnap.com/kb/absolute-path-vs-relative-path)**. 

EJM:

> Las ruta en la  cual nuestros scripts esperan encontrar los datos es en la carpeta del proyecto y dentro de  ``data/``

Cualquier otra duda, quedamos pendientes.

**Reply by galvanic90** [reply]
Hola @Felipe-RA, gracias por tu respuesta. Es que he visto varios data sets en github. Pero si tú me recomiendas que los cargue usando la api de Kaggle así lo haré! muchas gracias de nuevo!
